test_resolwe_runtime_utils.py

```diff
@@ -15,19 +15,48 @@
 # pylint: disable=missing-docstring
 from io import StringIO
 import json
+import os
+import shutil
 import sys
 from unittest import TestCase
 from unittest.mock import patch
 
+import responses
+import requests
+
 from resolwe_runtime_utils import (
-    save, export, save_list, save_file, save_file_list, save_dir, save_dir_list,
-    info, warning, error, progress, checkrc,
-    _re_save_main, _re_export_main, _re_save_list_main, _re_save_file_main, _re_save_file_list_main,
-    _re_save_dir_main, _re_save_dir_list_main, _re_info_main, _re_warning_main, _re_error_main,
-    _re_progress_main, _re_checkrc_main
+    save,
```

Comment:
This should probably be part of "black" commit.

Reply:
done